lib/Basic/TargetInfo.cpp

```diff
@@ -130,7 +134,18 @@ void targetinfo::printTripleInfo(
   out << "\",\n";
 
   out << "    \"platform\": \"" << getPlatformNameForTriple(triple) << "\",\n";
-  out << "    \"arch\": \"" << swift::getMajorArchitectureName(triple) << "\",\n";
+  out << "    \"arch\": \"" << swift::getMajorArchitectureName(triple)
+      << "\",\n";
+
+  clang::DiagnosticsEngine DE{new clang::DiagnosticIDs(),
+                              new clang::DiagnosticOptions(),
+                              new clang::IgnoringDiagConsumer()};
+  std::shared_ptr<clang::TargetOptions> TO =
+      std::make_shared<clang::TargetOptions>();
+  TO->Triple = triple.str();
+  clang::TargetInfo *TI = clang::TargetInfo::CreateTargetInfo(DE, TO);
+  out << "    \"pointerWidthInBits\": "
+      << TI->getPointerWidth(clang::LangAS::Default) << ",\n";
```

Comment:
If this is for `CMAKE_SIZEOF_VOID_P`, we should include the size in bytes since that's what CMake expects there.

Reply:
I think that the bitwidth is still nice to have. I've added an additional `pointerWidthInBytes`.